packages/devtools_app/test/fixtures/flutter_error_app/lib/main.dart

```diff
@@ -2,6 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+// @dart=2.9
```

Comment:
why are we "unmigrating" these files

Reply:
oh, seems to be result of merge. cleared